src/polyfills.js

```diff
@@ -35,7 +37,7 @@ import {getMode} from './mode';
   sure to not `install` it during dev since the `install` is done as a side
   effect in importing the module.
 */
-if (!getMode().localDev) {
+if (!getMode().localDev && !self.customElements) {
```

Comment:
so this is mostly a runtime path change right. module is still included but document-register-element no longer takes over if custom elements v1 is supported.

Reply:
Right. I might take it out, but the polyfill should now do nothing anyway.